docs/application/native/guides/data/data-storages.md

```diff
@@ -17,11 +17,20 @@ The main features of the Storage API include:
 
 ## Prerequisites
 
-To use the functions and data types of the Storage API (in [mobile](../../api/mobile/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) applications), include the `<storage.h>` header file in your application:
+To enable your application to use the storage functionality:
 
-```
-#include <storage.h>
-```
+1. To use the functions and data types of the Storage API (in [mobile](../../api/mobile/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__STORAGE__MODULE.html) applications), include the `<storage.h>` header file in your application:
+
+   ```
+   #include <storage.h>
+   ```
+
+2. To access files or directories in storage, you must declare following privileges:
+
+   - For the internal storage, declare `http://tizen.org/privilege/mediastorage`
```

Comment:
Suggestion:
This line is fine. However, to maintain consistency with other pages can we write as:
```
<privileges>
   <privilege>http://tizen.org/privilege/mediastorage</privilege>
</privileges>
```

For more information, see:
1. https://developer.tizen.org/development/guides/web-application/data-storage-and-management/stored-content-management
2. https://developer.tizen.org/development/guides/native-application/connectivity-and-wireless/bluetooth

Please verify and share your opinion.

@mijin85cho 
Please verify and share your valuable opinion regarding the same.

Reply:
Good checking, @Reni-90. 
The author updated the PR. Would you please check the latest?

Reply:
@mijin85cho 
Thank you. Sure, I will.